app/models/task.rb

```diff
@@ -50,6 +51,10 @@ def pending?
     !complete?
   end
 
+  def expired?
+    self.due_date && (self.due_date < Time.now && pending?)
```

Comment:
no need to wrap in parentheses, condition works in one direction

Reply:
I know it but it's more readable for me like "true && true" than "true && true && true"